nss-day-cohort-52/rare-api-smoky-pythons
# make migrations

### Issue ticket(s): #48 

### New file(s): None

### Modified File(s): All the model files 

### Modifications: Made sure that all foreign keys have a unique related_name argument

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to lz-migrations
3. Check to see if the migrations are successful
